src/Components/Profile/Address.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export const Address = () =>{
+
+    const AddressDetails= {
+        City: " new Cairo",
+        Street: "main north 90th street",
+        BlockNo:"45B",
+        Floor:" first "
+    }
+    return(
+        <>
+        <h3>Address</h3>
+        <div>
+            {Object.keys(AddressDetails).map( (key) => 
```

Comment:
Objects have no order, thus looping over their keys does not guarantee the same order of insertion, which is, in this case, is not a good user experience.